### DISCUSSION TITLE
Bitbucket Pipeline Image Digest Pinning

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#18611] [Request Help] [answered]
Hi all

Looking forward suggestions on the best way to enable digest pinning for bitbucket-pipeline.yml updates

**Comment by viceice** [comment, marked as answer]
Configure `pinDigests`[^1] for the manager and `depType=docker` via package rule.

[^1]: https://docs.renovatebot.com/configuration-options/#pindigests

**Reply by setchy** [reply]
As always, big thank you @viceice for your help.

This packageRule did the trick
```
"packageRules": [
    {
      "matchManagers": [
        "bitbucket-pipelines"
      ],
      "matchDepTypes": [
        "docker"
      ],
      "pinDigests": true
    }
  ]
  ```
